apps/persons/tests/test_cms_toolbars.py

```diff
@@ -0,0 +1,49 @@
+"""
+Test suite of the toolbar extension for person pages
+"""
+from django.test.utils import override_settings
+
+from cms.api import create_page
+from cms.test_utils.testcases import CMSTestCase
+from cms.toolbar.items import Menu, ModalItem
+
+from apps.core.factories import UserFactory
+from apps.courses.tests.utils import CheckToolbarMixin, get_toolbar_for_page
```

Comment:
The problem I see here is a cross-dependance between both apps. Because the courses application depends on the persons application to reference teachers... and now the persons application depends on the courses application for this test util.
So I think it is ok here to duplicate the code in both apps.
Or maybe put the util in the persons application and import it in the courses application?
Whichever you find the best approach